Content.Server/Mindshield/MindShieldSystem.cs

```diff
@@ -52,6 +61,17 @@ public void MindShieldRemovalCheck(EntityUid implanted, EntityUid implant)
         {
             _popupSystem.PopupEntity(Loc.GetString("head-rev-break-mindshield"), implanted);
             QueueDel(implant);
+            if (TryComp<BloodstreamComponent>(implanted, out var bloodstream))
+            {
+                _bloodstreamSystem.SpillAllSolutions(implanted, bloodstream);
+            }
+
+            if (HasComp<DamageableComponent>(implanted))
+            {
+                var damage = new DamageSpecifier(_Prototypes.Index<DamageGroupPrototype>("Genetic"), 300);
```

Comment:
don't hardcode the damage here. You should put this damage specifier either on a component somewhere or at the very least validate the prototype with a constant prototype Id validator in the system.

Reply:
I'm pretty sure I made the right changes here for what you mentioned, can you let me know if this is not correct. I moved it to a component and added that on to the MindShieldImplanter entity and reworked the system checks in a way that it could just be disabled back to current mechanic by removing that component off the MindShieldImplanter prototype.